unicodetools/src/main/resources/org/unicode/text/UCD/UnicodeInvariantTest.txt

```diff
@@ -1090,6 +1090,13 @@ Let $japaneseSimplifiedRadicals = \p{Name=/CJK RADICAL J-SIMPLIFIED/}
 In $chineseSimplifiedRadicals, Equivalent_Unified_Ideograph ∈ [\p{kRSUnicode=/^[0-9]+'\.0$/} $radicalsWithUnifiableSimplifications]
 In $japaneseSimplifiedRadicals, Equivalent_Unified_Ideograph ∈ \p{kRSUnicode=/^[0-9]+''\.0$/}
 
+# Egyptian hieroglyph invariants
+
+Let $unikemetScope = [\p{Block=/^Egyptian.Hieroglyphs/} - \p{gc=Cn}]
+$unikemetScope = [ [\p{gc=Lo} & \p{sc=Egyp}] - \p{Name=/^EGYPTIAN HIEROGLYPH (FULL |HALF |TALL |WIDE )?(BLANK|LOST SIGN)$/} ]
```

Comment:
just FYI: I always stumble a little over "=" meaning "assign" on a "Let" line but "assertEquals" on a test line (without a statement prefix) ... It would be a disruptive change, but using "==" for "assertEquals" might make this file more obvious.

Reply:
Or := (or ≔?) in `Let` (that would keep the actual invariant statements closer to normal mathematical operators on sets).

Not today, but good idea.

Reply:
Let with ASCII := would be ok